1_basics.ipynb

```diff
@@ -229,20 +453,44 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
-    "## 2. スクリーンキャストの埋め込み"
+    "## スクリーンキャストの埋め込み"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "## Screencast embedding"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "アニメーションgifを使った短時間のスクリーンキャストを埋め込むこともできます。\n",
     "\n",
     "以下の例では画面を10秒間録画し、notebook上に録画した結果を表示します。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "You can also embed a short screencast as animated gif.\n",
+    "\n",
+    "In the following example, the screen is recorded for 10 seconds and the recorded result is displayed on your notebook:"
```

Comment:
Ah, the expression is "in your notebook" not "on your notebook".
"On your notebook" would produce an image in the mind of something like "You put a cup of coffee **on** your notebook."  However, "You write about coffee **in** your notebook."

Funny, it is just a one letter difference. ;)
Here is the replacement sentence:

"In the following example, the screen is recorded for 10 seconds and the recorded result is displayed **in** your notebook:"

Reply:
Got it :)